search.html

```diff
@@ -29,29 +29,52 @@
                 </div>
             </nav>
         </header>
-
-        <div class="container">
-            <div class="row">
-                <!-- This is the wrapper for the aside and main -->
-                <aside id="container-aside">
-                    <div id="container-search">
-                        <form>
-                            <input type="text" id="search-input" name="search-input" placeholder="Movie Name">
-                        </form>
-                        <button id="btn-search" class="btn btn-search">Search</button>
-                    </div>
-                    <div id="container-recent">
-                        <h5>Recent</h5>
-                        <ul id="list-recent">
-
-                        </ul>
-                    </div>
-                </aside>
-                <button data-movie="ironman" id="btn-watch" class="btn btn-watch">Watch Trailer</button>
-
-                <a class="waves-effect waves-light btn">button</a>
+<!-- This the div for the entire main page -->
+<div class="container">
+    <div class="row">
+        <!-- This is the wrapper for the aside and main -->
+        <aside class="col s3" id="container-aside">
+            <div id="container-search">
+                <form>
+                    <input type="text" id="search-input" name="search-input" placeholder="Movie Name">
+                </form>
+                <button id="btn-search" class="waves-effect waves-light btn">Search</button>
+            </div>
+            <div id="container-recent">
+                <h5>Recent</h5>
+                <ul id="list-recent">
+                    <li><button>Movie Name Recent Search</button></li>
+                    <li><button>Movie Name Recent Search</button></li>
+                    <li><button>Movie Name Recent Search</button></li>
+                </ul>
+            </div>
+        </aside>
+        <div class="col s9">
```

Comment:
add an id = "resultCards" so that results can be hooked onto this <div> element

Reply:
This has been addressed